wagtail/admin/views/generic/multiple_upload.py

```diff
@@ -54,13 +54,13 @@ def get_edit_object_form_context_data(self):
         edit_form_class = self.get_edit_form_class()
         return {
             self.context_object_name: self.object,
-            "edit_action": reverse(self.edit_object_url_name, args=(self.object.id,)),
+            "edit_action": reverse(self.edit_object_url_name, args=(self.object.pk,)),
```

Comment:
Should we use `quote(self.object.pk)` so the primary key can contain characters like `/`? That's how Snippets (and modeladmin I think) do it currently

Reply:
Yep, I think we'd want to do that - but then we'd need to take care of unquoting them at the other end, and that's more untangling than I have time to do right now. (This PR is mostly about ensuring #6512 doesn't get perpetually version-bumped :-) )